Makefile

```diff
@@ -38,10 +38,10 @@ build:
 	touch .state/docker-build
 
 serve: .state/docker-build
-	docker-compose up
+	RECAPTCHA_SITE_KEY=$(RECAPTCHA_SITE_KEY) RECAPTCHA_SECRET_KEY=$(RECAPTCHA_SECRET_KEY) docker-compose up
```

Comment:
Are these actually needed? If ``RECAPTCHA_SECRET_KEY`` is defined in the environment, won't it automatically be inherited by the ``docker-compose up`` call?